reyna/src/com/b2msolutions/reyna/services/StoreService.java

```diff
@@ -38,8 +38,10 @@ public static void setCellularDataBlackout(Context context, TimeRange range) {
     }
 
     public static void setStorageSizeLimit(Context context, long limit) {
+        Logger.v(TAG, "setStorageSizeLimit, limit: " + limit);
         limit = limit < MINIMUM_STORAGE_LIMIT ? MINIMUM_STORAGE_LIMIT : limit;
 
+        Logger.v(TAG, "setStorageSizeLimit,  limit: " + limit);
```

Comment:
Should we put in the log mesage that the first setStorageSizeLimit is before comparing to min value and the second one after?